### ISSUE TITLE
Cannot add ScrollableCanvas from code

### ISSUE THREAD

**Issue by TungVuDuc2805** [Harley-xk/MaLiang#61]
I create a simple canvas view like this 
`    let canvas: ScrollableCanvas = {
        let sc = ScrollableCanvas(frame: .zero)
        sc.translatesAutoresizingMaskIntoConstraints = false
        return sv
    }()
`
and then add it fill to screen's view:

`view.addSubview(canvas)
        canvas.leadingAnchor.constraint(equalTo: self.view.leadingAnchor).isActive = true
        canvas.trailingAnchor.constraint(equalTo: self.view.trailingAnchor).isActive = true
        canvas.topAnchor.constraint(equalTo: self.view.topAnchor).isActive = true
        canvas.bottomAnchor.constraint(equalTo: self.view.bottomAnchor).isActive = true
`

But I cannot draw anything

**Comment by Harley-xk** [comment]
Yes, something is wrong here. Currently, set a non zero size to ScrollableCanvas when initializing will solve the problem.
I'll fix this issue at the bottom later.
